pontoon/tags/models.py

```diff
@@ -0,0 +1,12 @@
+
+from django.db import models
+
+from pontoon.base.models import Project, Resource
+
+
+class Tag(models.Model):
+    slug = models.CharField(max_length=20)
+    name = models.CharField(max_length=30)
+    project = models.ForeignKey(Project, blank=True, null=True)
+    resources = models.ManyToManyField(Resource)
+    priority = models.IntegerField(blank=True, null=True)
```

Comment:
We probably want to use choices here, e.g. like this:
https://github.com/mozilla/pontoon/blob/e65d7d59f301f499de144cc638bf2255c4236a23/pontoon/base/models.py#L1682